crates/check/tests/solution.rs

```diff
@@ -424,16 +422,15 @@ async fn predicate_graph_state_read() {
     let b_ca = content_addr(&b);
     let c_ca = content_addr(&c);
 
-    let node = |program_address, edge_start, reads| Node {
+    let node = |program_address, edge_start| Node {
```

Comment:
This test won't really work without reads so I suggest `#[ignore]` for now until we introduce the separate ops